index.html

```diff
@@ -16,31 +19,226 @@
             </div>
             <nav class="menuholder">
                 <img src="src/menuicon.png" alt="test">
-    
+
             </nav>
         </div>
     </header>
-    
+
     <section class="container2">
-        <div class="headerholder headmargin"> 
+        <div class="headerholder headmargin">
             <h1 class="head">I’m Davian<br>Glad to see<br> you!</h1>
         </div>
-        <p class="headtext headmargin">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="headtext headmargin">I’m a software developer! I can help you build a product , feature or website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
 
         <div class="supporttext headmargin">
             <b class="suptext">
                 LET'S CONNECT
             </b>
         </div>
-        
-        <ul class="social"> 
+
+        <ul class="social headmargin">
             <li><a href=""><img src="src/Vector.svg" alt=""></a></li>
             <li><a href=""><img src="src/Linkedinicon.svg" alt=""></a></li>
             <li><a href=""><img src="src/Vector.png" alt=""></a></li>
             <li><a href=""><img src="src/Vector2.svg" alt=""></a></li>
             <li><a href=""><img src="src/Vector3.svg" alt=""></a> </li>
         </ul>
-
     </section>
+
+    <div class="grid-container">
+        <section class="card1">
+            <section><img class="portfolio-image" src="src/Snapshoot Portfolio.png" alt=""></section>
+            <div class="p-layout">
+                <h1 class="portfolio-head">Tonic</h1>
+                <ul class="frame">
+                    <li>
+                        <h2 class="canopy">CANOPY</h2>
```

Comment:
- [ ] Good job with your project. I would have approved it right away but it's part of the main requirement to have the main container of this section with a `section` tag and each project preferably a `div` tag. Kindly reverse what you have done here and there should be nothing else to hold your project back from getting approved. Each project shouldn't be its own section.

Reply:
Okay thanks, i'll make the necessary changes